modules/xml/src/main/resources/META-INF/mule-module.properties

```diff
@@ -10,7 +10,22 @@ artifact.export.classPackages=org.mule.runtime.module.xml.config,\
                               org.mule.runtime.module.xml.transformer.wire,\
                               org.mule.runtime.module.xml.util,\
                               org.mule.runtime.module.xml.xpath,\
+                              com.ctc.wstx.api,\
```

Comment:
should we also be exporting the packages from the woodstox dependencies? (stax2-api)